dynawo/sources/Modeler/DataInterface/test/TestStateVariable.cpp

```diff
@@ -202,13 +202,16 @@ initializeModelNetwork(shared_ptr<DataInterface> data) {
   parametersSet->createParameter("LOAD_beta", 0.);
   parametersSet->createParameter("LOAD_isRestorative", false);
   parametersSet->createParameter("LOAD_isControllable", false);
-  if (!data->getNetwork()->getTwoWTransformers().empty() && data->getNetwork()->getTwoWTransformers()[0]->getRatioTapChanger()) {
+  if (!data->getNetwork()->getTwoWTransformers().empty() && (data->getNetwork()->getTwoWTransformers()[0]->getRatioTapChanger() ||
+           data->getNetwork()->getTwoWTransformers()[0]->getPhaseTapChanger())) {
     parametersSet->createParameter("TFO_t1st_THT", 9.);
     parametersSet->createParameter("TFO_tNext_THT", 10.);
     parametersSet->createParameter("TFO_t1st_HT", 11.);
     parametersSet->createParameter("TFO_tNext_HT", 12.);
-    parametersSet->createParameter("TFO_tolV", 13.);
   }
+  if (!data->getNetwork()->getTwoWTransformers().empty() && data->getNetwork()->getTwoWTransformers()[0]->getRatioTapChanger()) {
```

Comment:
We could create two boolean hasRatioTapChanger and hasPhaseTapChanger to avoid this complicated code